docs/source/markdown/podman-network-create.1.md

```diff
@@ -73,8 +73,9 @@ Set driver specific options.
 
 All drivers accept the `mtu` option. The `mtu` option sets the Maximum Transmission Unit (MTU) and takes an integer value.
 
-Additionally the `bridge` driver supports the following option:
+Additionally the `bridge` driver supports the following options:
 - `vlan`: This option assign VLAN tag and enables vlan\_filtering. Defaults to none.
+- `isolate`: This option isolates networks by blocking traffic between those that have this option enabled.
```

Comment:
Would it be more accurate to say "This option isolates this network by preventing it from accessing other Podman-managed networks. Defaults to *true*"?

Reply:
It defaults to false and only isolates traffic between the networks that have this option enabled so not all podman networks!

Reply:
the idea of isolating all networks sounds nice but yeah it would break existing setups. Maybe we can introduce a default true in a more major version release?

Reply:
The setting should be in containers.conf and then allow distributions to set the default.  We can then pick the default and switch it in Fedora 37 to make things more secure.

Reply:
Just flip the default in netavark when the the option is unset.

Reply:
When do you consider this safe to do?  Ever?

Reply:
The next major podman version sounds reasonable to me.